### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sugargoat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,20 +36,20 @@ This distribution includes cryptographic software. Your country may have restric
...
+| Target | Description |
+| :-- | :-- |
+| [`fog-ingest-server`](./fog/ingest/server) | Obliviously post-processes the blockchain to organize transaction outputs according to fog hints.|
+| [`fog-ledger-server`](./fog/ledger/server) | Obliviously serves ledger materials such as rings, merkle proofs, and key images, used to verify whether a Txo was spent, or to construct new transactions. |
+| [`fog-report-server`](./fog/report/server) | Provides the ingest enclave's Attestation Verification Report for transaction construction. |
+| [`fog-view-server`](./fog/view/server) | Obliviously serves the post-processed Txos to clients who wish to check their balance and construct new transactions. |
 
 ## Build Instructions
 
 The workspace can be built with `cargo build` and tested with `cargo test`. Either command will recognize the cargo `--release` flag to build with optimizations.
```
Comment (queenbdubs) [inline README.md:48]: Should we expand the `cargo test` instructions?  Adding a line about needing to call `service postgresql start` at the `./mob prompt` before calling `cargo test`...Does this still apply in this repo?

Comment (sugargoat) [reply README.md:48]: Yes - excellent point!

File: README.md [hunk 2]
```diff
@@ -63,69 +63,69 @@ cargo build
```
Comment (queenbdubs) [inline README.md:63]:
```suggestion
To run the unittest tests locally, you need to start a local postgres instance, then run the tests.  You can do this at the mob prompt:

```
# From the root of the cloned repository
./mob prompt

# Start postgres
service postgresql start

# Run the tests
cargo test
```
```

Comment (queenbdubs) [reply README.md:63]: hmm this did not do what I expected....It seems like a couple limitations of Github: you can't make a comment on a part of the commit that wasn't changed, and you also cant nest a code block within a suggestion.  Basically, I was suggesting the following raw text go right *before* the `## Overview line` (as the last paragraph of Build Instructions section)...


To run the unittest tests locally, you need to start a local postgres instance, then run the tests.  You can do this at the mob prompt:

```
# From the root of the cloned repository
./mob prompt

# Start postgres
service postgresql start

# Run the tests
cargo test
```

Comment (sugargoat) [reply README.md:78]: Done! Thank you!

**Review by eranrund — APPROVED** [review]

**Review by joekottke — COMMENTED** [review]
LGTM

**Review by voloshyn — APPROVED** [review]
LGTM

**Review by queenbdubs — APPROVED** [review]
nice!

**Pull request merged**